test/util/bundleUtils.test.ts

```diff
@@ -259,14 +268,29 @@ describe('bundleUtils', () => {
   });
 
   describe('createLibraryPackageBundle', () => {
-    it('returns a bundle including a Library and all dependent Libraries on valid input', async () => {
-      const bundle = await createLibraryPackageBundle({ id: 'LibraryWithDeps' }, {});
-      expect(bundle.resourceType).toEqual('Bundle');
-      expect(bundle.entry).toHaveLength(2);
-      expect(bundle.entry).toEqual(
+    it('returns bundle including a Library and all dependent Libraries on valid input', async () => {
+      const { libraryBundle } = await createLibraryPackageBundle({ id: 'LibraryWithDeps' }, {});
+      expect(libraryBundle.resourceType).toEqual('Bundle');
+      expect(libraryBundle.entry).toHaveLength(2);
+      expect(libraryBundle.entry).toEqual(
         expect.arrayContaining([{ resource: LIB_WITH_DEPS }, { resource: LIB_WITH_NO_DEPS }])
       );
     });
+
+    it('returns rootLibRef with url and version when both present', async () => {
+      const { rootLibRef } = await createLibraryPackageBundle({ id: 'LibraryWithDeps' }, {});
+      expect(rootLibRef).toEqual('http://example.com/LibraryWithDeps|0.0.1-test');
+    });
+
+    it('returns rootLibRef with just url when url present and version missing', async () => {
+      const { rootLibRef } = await createLibraryPackageBundle({ id: 'LibWithNoDeps' }, {});
+      expect(rootLibRef).toEqual('http://example.com/LibraryWithNoDeps');
+    });
+
+    it('returns rootLibRef with just url when url present and version missing', async () => {
```

Comment:
I think the test description here is a bit off. Maybe:
```suggestion
    it('returns rootLibRef with id when url and version missing', async () => {
```